docs/decisions/0011-fhirserver.md

```diff
@@ -0,0 +1,145 @@
+
+FHIR is a standardized data format that is used nationally to report health data. 
+
+A FHIR Server assists in bringing together data from various systems into one common, readable dataset. The FHIR Server itself does not do any analytical work, but aims to normalize public health data. In the case of this project, public health data will be reported from a variety of sources such as prisons, pharamacies, schools, and hospitals. A FHIR Server will ingest the records and become a single source of truth for a public health agency's data. 
+
+
+## Decision Drivers
+
+**Integration with Existing Systems** - The CDC has an existing cloud environment (Azure) so being able to work well with their existing system helps the development process. The team is working to get the first version into production so ease of maintainability and integration are important. 
+
+**Flexiblility** - The platform should support flexibility in at least two respects:
+
+- Data model: the data model should be extensible to account for special needs of STLTs. 
+- Functional features: the architecture should support a variety of data storage platforms to allow for optimal performance in a variety of use cases.
+
+
+**Support for a broad and complex data model** -  The solution needs to capture all relevant data in a structured way. Healthcare data can be complex due to the different sources of healthcare data being reported differently. 
+
+**Analtyics** -  The platform needs to support epidemiologists' analytical tools.
+
+**History and Logging** - Healthcare data modifications, additions, and deletions need to be tracked over time for historical logging.
+
+**Time to market** - Working with a solution that accelerates our time to market 
+
+**Performance** - Performance in terms of availability, speed, reliability must be considered to develop a solution. 
+
+## Considered Options
+
+### FHIR Server
+ 
+Many FHIR services in the cloud are PaaS which would reduce the maintenence burden on the team because the software upgrades, server maintenence would be up to the service. 
+
+Pros:
+- Less need for software/infrasturcutre developer resources will be needed 
+- Standardized FHIR data structure for all STLTs and the CDC to use
+  
+Cons:
+- FHIR data needs to be read all at once in order to filter data. So for example, if you all vax records in the FHIR server, you would need to pull all the json data, then filter. 
+- Because of the slower performance, additional considerations are needed to ensure consuming data is not slow.
+
+### RDBMS or Big Data solution
+
+Instead of using FHIR and FHIR Server, the team could develop a custom solution with a RDBMS or NoSQL option. The team would fully manage the infrastructure and architecture and communicate with STLTs how to use our custom data model.
+
+Pros:
+- With a RDBMS solution, data can be accessed via queries quickly, there is no need to manipulate data further for analytics purpose
```

Comment:
This is partly true, but we'd still need to create a predefined schema that could make certain queries slow if we don't anticipate the need for a particular index.